examples/Lorenz/Lorenz_example.jl

```diff
@@ -0,0 +1,381 @@
+# Import modules
+include("GModel.jl") # Contains Lorenz 96 source code
+
+# Import modules
+using Distributions  # probability distributions and associated functions
+#using StatsBase
+using LinearAlgebra
+#using StatsPlots
+using GaussianProcesses
+using Plots
+using Random
+using JLD2
+
+# Import Calibrate-Emulate-Sample modules
+using CalibrateEmulateSample.EnsembleKalmanProcesses
+using CalibrateEmulateSample.GaussianProcessEmulator
+using CalibrateEmulateSample.MarkovChainMonteCarlo
+using CalibrateEmulateSample.Observations
+using CalibrateEmulateSample.Utilities
+using CalibrateEmulateSample.ParameterDistributionStorage
+
+rng_seed = 4137
+Random.seed!(rng_seed)
+
+# Output figure save directory
+figure_save_directory = "/home/mhowland/Codes/CESPlots/figures/lorenz_96_dynamic/debug/T_180/"
+data_save_directory = "/home/mhowland/Codes/CESPlots/data/lorenz_96_dynamic/debug/T_180/"
+
+# Governing settings
```

Comment:
I think we want to reorganize slightly, Basically to separate the forward model from the parameter learning.
- Setup only that which is required to run the "truth", e.g all numerical parameters and "true" parameter values
- Run the truth run
- Then, set up the priors for recovering parameters of choice and initialize EKI etc. 
- initialize EKI ...

Reply:
It should be a bit better now but I'm open to future modifications.